src/main/frontend/components/block.cljs

```diff
@@ -14,6 +14,7 @@
             [frontend.components.lazy-editor :as lazy-editor]
             [frontend.components.svg :as svg]
             [frontend.components.macro :as macro]
+            [frontend.encrypt :as e]
```

Comment:
In logseq,  `e` could be `event` or `error`, so using `encrypt` might be easier to understand than `e` here.